AppCenterCrashes/AppCenterCrashes/WrapperSDKUtilities/MSACWrapperCrashesHelper.h

```diff
@@ -14,35 +14,27 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * This general class allows wrappers to supplement the Crashes SDK with their own behavior.
  */
+NS_SWIFT_NAME(WrapperCrashesHelper)
 @interface MSACWrapperCrashesHelper : NSObject
 
 /**
- * Sets the crash handler setup delegate.
+ * Sets or gets the crash handler setup delegate.
  *
- * @param delegate The delegate to set.
  */
-+ (void)setCrashHandlerSetupDelegate:(id<MSACCrashHandlerSetupDelegate>)delegate;
+@property(class, nonatomic, setter=setCrashHandlerSetupDelegate:, getter=getCrashHandlerSetupDelegate) id<MSACCrashHandlerSetupDelegate>
```

Comment:
Please read style guidelines for getters..

Wait, it was already there in this form.. anyway, let's do it properly